packages/mobile/src/screens/personal-info/components/profile-image/profile-image.tsx

```diff
@@ -1,31 +1,95 @@
-import React, { FC } from 'react';
+import React, { FC, useState, useCallback } from 'react';
+import { image as imageService, notification } from '~/services/services';
 import {
   View,
-  UserIcon,
   Pressable,
   CameraIcon,
   FlagBackgroundView,
+  PhotoPickerModal,
 } from '~/components/components';
 import { globalStyles } from '~/styles/styles';
+import {
+  requestExternalStoragePermission,
+  pickImageCamera,
+  pickImageLibrary,
+} from '~/helpers/helpers';
+import { UserAvatar } from '../avatar/avatar';
 import { styles } from './styles';
 
 const ProfileImage: FC = () => {
+  const [showModal, setShowModal] = useState(false);
+  const [photoUri, setPhotoUri] = useState('');
+  const [IsUploading, setIsUploading] = useState(false);
+
+  const toggleModal = () => {
+    return setShowModal((showModal) => !showModal);
```

Comment:
Do not redeclare outer scope's variables
```suggestion
    return setShowModal((currentShowModal) => !currentShowModal);
```